src/components/stepforms/DomainStepForm.vue

```diff
@@ -40,7 +40,7 @@ export default class DomainStepForm extends BaseStepForm<DomainStep> {
 
   @State domains!: string[];
 
-  readonly title: string = 'Change your domain';
+  readonly title: string = 'Select a domain';
```

Comment:
We open the DomainStep form only for the edition, I think for choosing it we've got a special CTA

Reply:
Actually "Change" sounds less natural English to my ears in such a context. To say "Change your domain" I would rather say "Select a new domain" but I am not an linguistic expert !!

Also, I may decide not to "change" the domain, I can leave it unchanged eventually. The 100% truth is in the selection, not the change :)

And my last answer will be: we are not searching for an important CTA like a payment button, so I think we can be quite flexible on synonyms ;)

Reply:
👍